Jenkinsfile

```
@@ -1,4 +1,9 @@
 #!/usr/bin/env groovy
 
 /* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin()
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 21],
```

```suggestion
    [platform: 'linux', jdk: 21],
    [platform: 'linux', jdk: 17],
```

JDK 17 is still on [premier support](https://endoflife.date/oracle-jdk).